airbyte-integrations/connectors/source-amazon-seller-partner/acceptance-test-config.yml

```diff
@@ -48,6 +48,72 @@ tests:
 #      configured_catalog_path: "integration_tests/configured_catalog_browse_tree_data.json"
 #    - config_path: "secrets/config.json"
 #      configured_catalog_path: "integration_tests/configured_catalog_restock_inventory.json"
+#    - config_path: "secrets/config.json"
+#      timeout_seconds: 3600
```

Comment:
Remove `timeout_settings` here and for all other streams since many streams require manual adjustments to get them passed. It's better to get errors early and then look closely at the failing tests.

```suggestion
```